std_msgs/Header header
uint32 state
uint32 image_width
uint32 image_height
uint32 focal_lenth_x
uint32 focal_lenth_y
uint32 radius
geometry_msgs/Point tar
uint32 circle_activate
uint32 circle_terminate
